package.json

```
@@ -26,13 +26,23 @@
     "typecheck:shared": "tsc --noEmit --project packages/shared",
     "typecheck:frontend": "tsc --noEmit --project packages/frontend",
     "typecheck:backend": "tsc --noEmit --project packages/backend",
-    "typecheck": "npm run typecheck:shared && npm run typecheck:frontend && npm run typecheck:backend"
+    "typecheck": "npm run typecheck:shared && npm run typecheck:frontend && npm run typecheck:backend",
+    "prepare": "husky install"
   },
   "resolutions": {
     "string-width": "^4.2.0",
     "wrap-ansi": "^7.0.0"
   },
+  "lint-staged": {
+    "*.{js,ts,tsx,svelte}": [
+      "node --max-old-space-size=6144 node_modules/eslint/bin/eslint.js --cache --fix",
```

in podman desktop we're requiring much more memory else it tails but AFAIK here it can just be `eslint --cache --fix`
```suggestion
      "eslint --cache --fix",
```

and if you introduce caching, maybe need (other PR) to introduce caching in existing `lint:check` and `lint:fix` rules

---

It's only ~5s, so I just removed the caching entirely, much simpler.